client/landing/gutenboarding/gutenboard.tsx

```diff
@@ -40,20 +40,26 @@ export function Gutenboard() {
 	// TODO: Explore alternatives for loading fonts and optimizations
 	// TODO: Don't load like this
 	useEffect( () => {
-		fontPairings.forEach( pair =>
-			Object.keys( pair ).forEach( k => {
-				const fontKey = k as keyof FontPair;
-				const { title, fontFamily }: { title: string; fontFamily: string } = pair[ fontKey ];
-				const isBold = fontKey === 'headings';
-				const l = document.createElement( 'link' );
-				l.href = `https://fonts.googleapis.com/css2?family=${ encodeURI(
-					`${ fontFamily }${ isBold ? ':wght@700' : '' }`
-				) }&text=${ encodeURI( title ) }&display=swap`;
-				l.rel = 'stylesheet';
-				l.type = 'text/css';
-				document.head.appendChild( l );
-			} )
-		);
+		fontPairings.forEach( ( { base, headings } ) => {
+			const linkBase = document.createElement( 'link' );
+			const linkHeadings = document.createElement( 'link' );
+
+			linkBase.href = `https://fonts.googleapis.com/css2?family=${ encodeURI(
+				base
+			) }&text=${ encodeURI( getFontTitle( base ) ) }&display=swap`;
+			linkHeadings.href = `https://fonts.googleapis.com/css2?family=${ encodeURI(
```

Comment:
I really think we should create a helper function that constructs the font URL. This way, if we migrate away from Google fonts, we modify one place.

Reply:
Yep, this is something we'll probably want down the line. I'll create a backlog issue.

Reply:
#40613